.github/CHANGELOG.md

```diff
@@ -2,6 +2,76 @@
 
 <h3>New features since last release</h3>
 
+* The `RotosolveOptimizer` now can tackle more general parametrized circuits using trigonometric
+  interpolation as discussed in [Vidal and Theis, 2018](https://arxiv.org/abs/1812.06323)
+  and in [Wierichs, Izaac, Wang, Lin 2021](https://arxiv.org/abs/2107.12390).
+  [(#1489)](https://github.com/PennyLaneAI/pennylane/pull/1489)
+
+  Consider a circuit with a mixture of Pauli rotation gates, controlled Pauli rotations and
+  single-parameter layers of Pauli rotations:
+  ```python
+  dev = qml.device('default.qubit', wires=3, shots=None)
+
+  @qml.qnode(dev)
+  def cost_function(rot_param, layer_par, crot_param):
+      for i, par in enumerate(rot_param):
+          qml.RX(par, wires=i)
+      for w in dev.wires:
+          qml.RX(layer_par, wires=w)
+      for i, par in enumerate(crot_param):
+          qml.CRY(par, wires=[i, (i+1)%3])
+
+      return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1) @ qml.PauliZ(2))
+  ```
+  Then this cost function supports one frequency for each of the first `RX` rotation angles,
+  three frequencies for the layer of `RX` gates that depend on `layer_par`, and two
+  frequencies for each of the `CRY` gate parameters. Rotsolve can then be used to minimize
+  the `cost_function`:
+
+  ```python
+  # Initial parameters
+  init_param = [
+      np.array([0.3, 0.2, 0.67], requires_grad=True),
+      np.array(1.1, requires_grad=True),
+      np.array([-0.2, 0.1, -2.5], requires_grad=True),
+  ]
+  # Numbers of frequencies per parameter
+  num_frequencies = [[1, 1, 1], 3, [2, 2, 2]]
```

Comment:
There seems to be the assumption here that the frequencies are always the first R integers. Should this be made more explicit?